src/labels/CustomLabels.labels

```diff
@@ -2752,6 +2752,14 @@ To check your GAU Allocation settings, go to Donations | GAU Allocations.</value
         <value>- Household Duplicate</value>
     </labels>
     <labels>
+        <fullName>hhErrorInvalidContactFieldName</fullName>
+        <categories>Households, Household Naming</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>hhErrorInvalidContactFieldName</shortDescription>
+        <value>The field, {0}, does not match any field name on Contact.</value>
```

Comment:
I just noticed that within the Settings page, you do prefix the error with "Error in Informal Greeting Format: ..." so can we also include the appropriate prefix in other scenarios?